src/NHSD.BuyingCatalogue.Identity.Api/Styles/login.scss

```diff
@@ -9,3 +9,7 @@
 .bc-c-login-page-error-box-padding {
     white-space: pre-wrap;
 }
+
+.input-validation-error {
```

Comment:
have you used this class anywhere?

Reply:
Yes, it gets used when there is an error on the input. (If you launch the login page, then submit with an error, inspect the input box)

Reply:
After speaking with @kat1906 going to use the nhsuk-input--error class